### PR TITLE
Submitting Munixo Client version 5.0.65.14161

### PR DESCRIPTION
**Pull request by tamas-huszar**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- yes
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- yes
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- yes, validatin was ok
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- yes, winget must be run as administrator (or msi install requires it)
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?
- yes, using the minimal singleton format

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Submitting Munixo Client, more info about the software is available at https://www.novicon.net/munixo.html

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/55853)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
It appears your file is encoded in binary; Please ensure that it is encoded in UTF8

I recommend using wingetcreate or YamlCreate to ensure your manifest is encoded with the right character set

File: manifests/n/Novicon/Munixo/Client/5.0.65.14161/Novicon.Munixo.Client.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+PackageIdentifier:            Novicon.Munixo.Client
+PackageVersion:               5.0.65.14161
+PackageLocale:                en-US
+Publisher:                    Novicon
+PackageName:                  munixo Client
+License:                      Copyright (C) 2022 Novicon GmbH
```
Comment (Trenly) [inline manifests/n/Novicon/Munixo/Client/5.0.65.14161/Novicon.Munixo.Client.yaml:6]:
```suggestion
Copyright:                      Copyright (C) 2022 Novicon GmbH
```
A copyright is not a license; Please add the appropriate license terms

File: manifests/n/Novicon/Munixo/Client/5.0.65.14161/Novicon.Munixo.Client.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+PackageIdentifier:            Novicon.Munixo.Client
+PackageVersion:               5.0.65.14161
+PackageLocale:                en-US
+Publisher:                    Novicon
```
Comment (Trenly) [inline manifests/n/Novicon/Munixo/Client/5.0.65.14161/Novicon.Munixo.Client.yaml:4]:
```suggestion
Publisher: Novicon GmbH
```
This must match what is shown in control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/n/Novicon/Munixo/Client/5.0.65.14161/Novicon.Munixo.Client.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+InstallerSwitches:
+ Silent:                      /qn
+ SilentWithProgress:          /q
+ InstallLocation:             APPDIR=<INSTALLPATH>
+ Log:                         /lv <LOGPATH>
+Installers: 
+ - Architecture:              x64
+   InstallerUrl:              https://update.munixo.com/client_windows_stable/munixo-Client.5.0.65.14161.msi
+   InstallerSha256:           2DD59F37C860ADC2434571C37BDE50F68E756B8138448D431E975501EA27778B
+ManifestType:                 singleton
```
[the remark this answers is not in the document]
Comment (tamas-huszar) [reply manifests/n/Novicon/Munixo/Client/5.0.65.14161/Novicon.Munixo.Client.yaml]: I resolved all requested changes in commit [4ba8ddb](https://github.com/microsoft/winget-pkgs/pull/55853/commits/4ba8ddb18a293cd0e855eda839ceb69e2a1aa2bc)

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**